src/lib/draw-and-pick.js

```diff
@@ -196,6 +210,17 @@ export function pickLayers(gl, {
     if (info) {
       infos.set(info.layer.id, info);
     }
+
+    const pickingSelectedColor = pickedColor;
+    const pickingValid = Boolean(
```

Comment:
This is a lot of logic required to use the `picking` module. Makes me feel that the `picking` module API design was not successful. I wonder if we could do anything to simplify?